src/sentry/search/events/builder/metrics.py

```diff
@@ -136,6 +137,10 @@ def use_default_tags(self) -> bool:
                 self._use_default_tags = False
         return self._use_default_tags
 
+    @property
+    def use_next_spec_version(self) -> bool:
+        return features.has(NEXT_SPEC_FEAT_FLAG, self.params.organization, actor=None)
```

Comment:
We may need to set the `actor` value to something to allow our emails in Flagr to do something useful.